tox.ini

```diff
@@ -8,8 +8,7 @@ skip_missing_interpreters=True
 [testenv]
 deps =
     -rtest-requirements.txt
-    py35-flake8: flake8-bugbear
-    ansible22: ansible~=2.2
```

Comment:
Seems likely that's the problem. I thought it was superfluous with ansible specified in the requirements.txt. Apparently not. Will fix.